specification/security/resource-manager/Microsoft.Security/preview/2020-01-01-preview/secureScore.json

```diff
@@ -0,0 +1,615 @@
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SecureScoreControlsDefinitionList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SecureScoresList": {
+      "type": "object",
+      "description": "Page of a secure scores list",
+      "properties": {
+        "value": {
+          "description": "Collection of secure scores in this page",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SecureScoresItem"
+          }
+        },
+        "nextLink": {
+          "description": "The URI to fetch the next page.",
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "SecureScoresItem": {
+      "description": "Secure score item data model",
+      "readOnly": true,
+      "type": "object",
+      "properties": {
+        "properties": {
+          "description": "Secure score item",
+          "readOnly": true,
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SecureScoreItemProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "SecureScoreItemProperties": {
+      "description": "Describes properties of a calculated secure score.",
+      "readOnly": true,
+      "type": "object",
+      "properties": {
+        "displayName": {
+          "description": "User friendly display name of the secure score item",
+          "readOnly": true,
+          "type": "string"
+        },
+        "score": {
+          "description": "score object",
+          "readOnly": true,
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SecureScoreScoreDetails"
+        }
+      }
+    },
+    "SecureScoreScoreDetails": {
+      "description": "Calculation result data",
+      "readOnly": true,
+      "type": "object",
+      "properties": {
+        "max": {
+          "description": "Maximum score applicable",
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32"
+        },
+        "current": {
+          "description": "Actual score",
+          "readOnly": true,
+          "type": "number",
+          "format": "double"
+        }
+      }
+    },
+    "SecureScoreControlScoreObject": {
```

Comment:
why do you have 2 objects that look exactly the same?

Reply:
One with range of 0..10
The other is an accumulation of all single items